p/api/greader.php

```diff
@@ -104,17 +108,15 @@ function debugInfo(): string {
 
 final class GReaderAPI {
 
-	/** @return never */
-	private static function badRequest() {
+	private static function badRequest(): void {
```

Comment:
I think this is not the intend. We should use `never` as a type-hint but it is not available until PHP 8.1. The same applies for other methods ending with a `die`. Maybe this applies also for methods ending with an `exit`.

Reply:
I understand. I can't wait for the release of the new version of debian, we will be able to type for php 7.4 at least

Reply:
done